test/string.js

```diff
@@ -3180,6 +3180,42 @@ describe('string', () => {
             ], done);
         });
 
+        it('validates an base64 string with explicit padding required', (done) => {
+
+            const rule = Joi.string().base64(true);
+            Helper.validate(rule, [
+                ['YW55IGNhcm5hbCBwbGVhc3VyZS4=', true],
```

Comment:
We should also include a test for 2 bytes of padding (`==` instead of `=`). ie. `YW55IGNhcm5hbCBwbGVhc3VyZQ==`